proc.go

```diff
@@ -95,6 +95,10 @@ func (p Proc) CmdLine() ([]string, error) {
 	if err != nil {
 		return nil, err
 	}
+	
+	if len(data) < 1 {
+		return nil, nil
```

Comment:
I think we should return `[]string{}, nil` here, as the assumption in go for `type, error` return signatures is usually: if `err` is `nil` then the returned type is an initialized value.